Functions/Sine.py

```diff
@@ -0,0 +1,35 @@
+#import math
+import Power_Function as Pwr
+from Constants import get_pi
+"""
+    Names of functions are "sine" and "factorial"
+    Each only takes 1 input which is the number
+"""
+
+
+def factorial(number):
+    # factorial by recursion
+    if number < 0:
+        print("factorial is undefined for negative numbers!")
+        return None
+    if number == 1 or number == 0:
+        return 1
+    else:
+        return number*factorial(number-1)
+
+
+def sine(num):
```

Comment:
This needs to take a variable number of arguments otherwise it won't be compatible with the CLI. The function needs to handle throwing exceptions up to main too if it's passed too many or too little arguments within the list. Check out the power_function for how to implement it